repositories.yaml

```
@@ -186,6 +199,7 @@ applications:
       - org.mozilla.appservices:logins
       - org.mozilla.components:support-migration
       - org.mozilla.components:places
+      - org.mozilla.appservices:nimbus
```

Assuming you want `org.mozilla.components`
```suggestion
      - org.mozilla.components:nimbus
```

---

This was a bit of a mystery to me, nimbus has moved to the appservices repo now and it soon may only have a shell in android-components, but that is the place we want it to be consumed from.

---

Honestly it doesn't matter, you could just call it `nimbus` and it would be fine. We just use these strings for dependency lookups downstream from the probe-info-service.

---

"It doesn't matter" in that _what_ the string is doesn't matter to anyone outside the probe-info-service. It still needs to match between `dependency_name` and `dependencies`.

---

So it should probably be `org.mozilla.components:service-nimbus

---

> "It doesn't matter" in that _what_ the string is doesn't matter to anyone outside the probe-info-service. It still needs to match between `dependency_name` and `dependencies`.

Gotcha, let's keep it simple and just `nimbus` then, if that's okay with you?

---

FWIW the glean dictionary now displays some dependency information (and will possibly soon display more) so it might be good to keep any conventions used consistent: https://dictionary.protosaur.dev/apps/fenix/app_ids/org_mozilla_fenix

---

@wlach it might be good to list the friendly name of the dependencies rather than the dependency_name string, what do you think?

---

I think I'm onboard with that: we should think about this holistically in terms of what we want the UX in the glean dictionary to be. I suspect the key user story is helping people (likely data engineers and application developers) better understand where metrics for a given application are coming from. In that respect I think the metadata for the library itself (rather than the references to it in repositories.yaml) is most important.